pkg/querier/queryrange/instrument.go

```diff
@@ -0,0 +1,47 @@
+package queryrange
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/grafana/loki/pkg/querier/queryrange/queryrangebase"
+)
+
+const (
+	gRPC = "gRPC"
+)
+
+type QueryHandlerMetrics struct {
+	InflightRequests *prometheus.GaugeVec
+}
+
+func NewQueryHandlerMetrics(registerer prometheus.Registerer, metricsNamespace string) *QueryHandlerMetrics {
+	return &QueryHandlerMetrics{
+		InflightRequests: promauto.With(registerer).NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: metricsNamespace,
```

Comment:
should we set this to `loki`?

`metricsNamespace` was added for migrating metrics that were using `cortex` prefix. But in this case it was already set to `loki` I believe, so there is no need to make the prefix configurable now

Reply:
I think it should stay flexible but thanks for the notice.